### PR TITLE
add icon and favicon for armsinstitutewiki per request T1757

### PR DESCRIPTION
**Pull request by videojeux4**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -4757,6 +4757,7 @@
 		'anothertimeline2120wiki' => "//$wmgUploadHostname/anothertimeline2120wiki/6/64/Favicon.ico",
 		'applewikiwiki' => "//$wmgUploadHostname/applewikiwiki/8/89/Facveicon.ICO",
 		'applebranchwiki' => "//$wmgUploadHostname/applebranchwiki/e/ee/AppleBranch_Icon.png",
+		'armsinstitutewiki' => "//$wmgUploadHostname/armsinstitutewiki/6/64/Favicon.ico"
```
Comment (Reception123) [inline LocalSettings.php]: You are missing a comma

File: LocalSettings.php [hunk 2]
```diff
@@ -4896,6 +4897,7 @@
 		'applebranchwiki' => "//$wmgUploadHostname/applebranchwiki/0/03/AppleBranch_135.png",
 		'applewikiwiki' => "//$wmgUploadHostname/applewikiwiki/c/c9/AppleWikilogo.PNG",
 		'arefepvawiki' => "//$wmgUploadHostname/arefepvawiki/9/92/Logoaf.png",
+		'armsinstitutewiki' => "//$wmgUploadHostname/armsinstitutewiki/2/2d/Logo-arms-institute-160.png"
```
Comment (Reception123) [inline LocalSettings.php]: Same as above

File: LocalSettings.php (hunk 1 above)
Comment (Reception123) [inline LocalSettings.php]: (At the end)

**Comment by videojeux4** [comment]
Yes i view this on travis i have corrected :)

**Pull request merged**